### ISSUE TITLE
Feature Request: Display Warning on Redirects

### ISSUE THREAD

**Issue by jvanderaa** [jeffkala/pyurlcheck#19]
I went looking to check on the code about what happens with a redirect. I believe it would be worth while to display a warning when there is a 3xx redirect that happens. This will allow for the proper link to get updated, but the content is still being displayed.

**Comment by jeffkala** [comment]
@jvanderaa thanks for the feature request.  Will look into options.

**Comment by jeffkala** [comment]
@jvanderaa was playing with some options and ran a test that gives this output.  Is this helpful?  Or would it be more helpful if https redirects and index page redirects were not included?

```
WARNING: https://www.readthedocs.io had redirects while executing. Redirects are https://readthedocs.org/!
WARNING: http://google.com had redirects while executing. Redirects are http://www.google.com/ => https://www.google.com/?gws_rd=ssl!
WARNING: https://youtube.com had redirects while executing. Redirects are https://www.youtube.com/!
WARNING: http://facebook.com had redirects while executing. Redirects are https://facebook.com/ => https://www.facebook.com/!
WARNING: http://google.com had redirects while executing. Redirects are http://www.google.com/ => https://www.google.com/?gws_rd=ssl!
WARNING: https://youtube.com had redirects while executing. Redirects are https://www.youtube.com/!
WARNING: http://facebook.com had redirects while executing. Redirects are https://facebook.com/ => https://www.facebook.com/!
WARNING: http://resources.intenseschool.com/gns3-lab-introduction-to-dmvpn had redirects while executing. Redirects are http://resources.intenseschool.com/gns3-lab-introduction-to-dmvpn/
```

**Comment by jeffkala** [comment]
Check out develop branch to see the enhancement.

**Comment by jeffkala** [comment]
Should be fixed in #21

**Comment by jvanderaa** [comment]
Looking good!

**Comment by jeffkala** [comment]
fixed in #21
